builtin/credential/ldap/path_config.go

```diff
@@ -100,6 +106,10 @@ func (b *backend) pathConfigWrite(
 	if groupdn != "" {
 		cfg.GroupDN = groupdn
 	}
+	sslverify := d.Get("sslverify").(bool)
+	if !sslverify {
+		cfg.SSLVerify = sslverify
```

Comment:
Would a warning in the help output be enough? I'm looking in the code now and I don't see a great spot to put an output on the actual write. In testing I changed pathConfigWrite() to send back a logical.Response even on success but that doesn't appear to be super clean as the actual "Success!" write message then gets trampled on.
I did update the code to make ignoring SSL verification non-default.